### DISCUSSION TITLE
Backup from online Dropbox to local external hdd

### DISCUSSION THREAD

**Discussion by Morse6690** [iterate-ch/cyberduck#16204] [Q&A]
I am using a win 11 pc and just removed the Dropbox app from my system. Previously I backed up my Dropbox data regularly from the local Dropbox folder to my local external drive using bvckup2 or free file sync. The vaults were closed/locked when backing up, so the whole vault was backed up. 
My plan in purchasing mountain duck was that a similar provess would be possible.
Problems.
1. When using MD, I have saved the password for a vault to make opening the files in said vault easier. However when running a backup job, the real file names are shown as being backed up. I believe this is because this vault is automatically opened by MD. 
Is there any way to lock the vault in MD before the backup is run? 
2. The file count is different when I compare (1) backing up from the local Dropbox folder VS (2) backing up from MD. 
3. Also the open vault has all the files backed up unencrypted. I guess this could explain the difference in file count. 

Is there any way I could improve this work flow and back up all vaults while locked and encrypted? 
Is this even possible to achieve?

**Comment by tatze96** [comment]
Correct. If you have automatic vault detection activated, Mountain Duck will unlock the Cryptomator vault by itself as long as the password is saved within Keychain. So, if you deactivate the vault detection within the Mountain Duck preferences _Vault_ tab, the backup shouldn't show the decrypted files.

Note that you'll have to unlock the vault manually through the context menu when needed if you have the automatic vault detection deactivated.

Regarding the difference in file count:
The file count differs depending on whether the vault is currently locked or decrypted. The locked or encrypted view of a vault generally contains more items than the decrypted one.
